src/nvim/os/env.c

```diff
@@ -429,6 +411,7 @@ void init_homedir(void)
   homedir = NULL;
 
   const char *var = os_getenv("HOME");
+  const char *setmefree = var;
```

Comment:
`must_free` seems like the most common name in the codebase
```suggestion
  const char *must_free = var;
```

Reply:
> However, don't you agree that it may be conceptually beneficial to distinguish between `bool` and `const [char|int|..] *` deallocation helper variables?

That's a good thought, but unless both kinds are in the same function body, risk of ambiguity/confusion is very low.

Reply:
`tofree`